schemas/external/repo/asset.schema.json

```diff
@@ -38,6 +45,17 @@
           "meta:usereditable": false,
           "description": "The version ID of the piece of content. It will be generated both on explicit and implicit save or upload."
         },
+        "repo:createdBy": {
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "ID of the user who initiated the action that caused the resource to be created in the repository."
+        },
+        "repo:modifiedBy": {
+          "type": "string",
+          "meta:usereditable": false,
+          "description": "ID of the user who initiated the action that most recently caused the resource to be modified in the repository."
+        },
         "tiff:imageWidth": {
```

Comment:
Why is this in asset and not in image?  Same with imageLength

Reply:
I don't know. This PR preserves the existing definition, and changing this definition is outside the scope of this change. I respectfully suggest that any further discussion on this point is deferred to a separate PR.